src/native/api.h

```diff
@@ -3,6 +3,8 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
+  
```

Comment:
Nitpicking: there is a trailing whitespace on this line. It would be great to get rid of it :)

Reply:
@indutny thanks, let me do:)